docs/backplane/requirements/cee/20-machineautoscaler.md

```diff
@@ -0,0 +1,22 @@
+# CEE on OSD/ROSA
+
+label selectors:
+
+- managed=true (implied)
+
+These premissions are the baseline across the managed fleet and apply to all product flavors:
+
+- OSD
+- ROSA Classic
+- ROSA HCP
```

Comment:
```suggestion
```

HCP does not expose autoscaling, this has to be managed / viewed via OCM or on management cluster hosting the control plane.